code/modules/reagents/chemistry/recipes.dm

```diff
@@ -21,15 +21,30 @@
 
 	///Determines if a chemical reaction can occur inside a mob
 	var/mob_react = TRUE
-	///Required temperature for the reaction to begin
-	var/required_temp = 0
-	/// Set to TRUE if you want the recipe to only react when it's BELOW the required temp.
-	var/is_cold_recipe = FALSE
+	
 	///The message shown to nearby people upon mixing, if applicable
 	var/mix_message = "The solution begins to bubble."
 	///The sound played upon mixing, if applicable
 	var/mix_sound = 'sound/effects/bubbles.ogg'
 
+	/// Set to TRUE if you want the recipe to only react when it's BELOW the required temp.
+	var/is_cold_recipe = FALSE
+	///Required temperature for the reaction to begin, for fermimechanics it defines the lower area of bell curve for determining heat based rate reactions, aka the minimum
+	var/required_temp = 100
+	///FermiChem! - See fermi_readme.md
+	var/optimal_temp = 500 // Upper end for above (i.e. the end of the curve section defined by temp_exponent_factor)
+	var/overheat_temp = 900 // Temperature at which reaction explodes - If any reaction is this hot, it explodes!
+	var/optimal_pH_min = 5 // Lowest value of pH determining pH a 1 value for pH based rate reactions (Plateu phase)
+	var/optimal_pH_max = 9 // Higest value for above
+	var/determin_pH_range = 4 // How far out pH wil react, giving impurity place (Exponential phase)
+	var/temp_exponent_factor = 2 // How sharp the temperature exponential curve is (to the power of value)
+	var/pH_exponent_factor = 1 // How sharp the pH exponential curve is (to the power of value)
+	var/thermic_constant = 1 // Temperature change per 1u produced
+	var/H_ion_release = 0.01 // pH change per 1u reaction
+	var/rate_up_lim	= 20 // Optimal/max rate possible if all conditions are perfect
+	var/purity_min = 0.15 // If purity is below 0.15, it calls OverlyImpure() too. Set to 0 to disable this.
+	var/reactionFlags // bitflags for clear conversions; REACTION_CLEAR_IMPURE, REACTION_CLEAR_INVERSE, REACTION_CLEAR_RETAIN, REACTION_INSTANT
```

Comment:
Same as suggested before, use the dmdocs format with `///` on the line over the variable.
If you want more detailed information:
<https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md>

Reply:
That should be the last camel case done, let me know if I missed any.